types/chrome/index.d.ts

```diff
@@ -6884,11 +6892,11 @@ declare namespace chrome.runtime {
         // Optional
         action?: ManifestAction | undefined;
         background?:
-            | {
-                  service_worker: string;
-                  type?: 'module'; // If the service worker uses ES modules
-              }
-            | undefined;
+        | {
+            service_worker: string;
+            type?: 'module'; // If the service worker uses ES modules
+        }
+        | undefined;
```

Comment:
Undo the formatting change
```suggestion
            | {
                  service_worker: string;
                  type?: 'module'; // If the service worker uses ES modules
              }
            | undefined;```